### PR TITLE
cc_keyboard: add Alpine support

### PR DESCRIPTION
**Pull request by dermotbradley**
## Proposed Commit Message

```
cc_keyboard: add Alpine support

Add support for Alpine Linux to cc_keyboard.
```

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/development/contributing.html)
 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: cloudinit/distros/alpine.py [hunk 1]
```diff
@@ -103,6 +107,20 @@ def _read_hostname(self, filename, default=None):
...
 
+    def set_keymap(self, layout, model, variant, options):
+        if not layout:
+            LOG.error("Keyboard layout not specified.")
+            return
+        if not variant:
+            LOG.error("Keyboard variant not specified.")
+            return
+        if model:
+            LOG.info("Keyboard model is ignored for Alpine Linux.")
```
Comment (TheRealFalcon) [inline cloudinit/distros/alpine.py:118]: If somebody passed config expecting it to do something, I think it's worth a warning (same with 2 lines down).

File: tests/unittests/config/test_cc_keyboard.py [hunk 2]
```diff
@@ -73,5 +76,46 @@ def test_schema_validation(self, config, error_msg):
...
+    @mock.patch("cloudinit.distros.subp.subp")
+    def test_alpine_linux_cmd(self, m_subp, *args):
+        """Alpine Linux runs setup-keymap"""
+        cfg = {"keyboard": {"layout": "us", "variant": "us"}}
+        layout = "us"
+        variant = "us"
+        cloud = get_cloud("alpine")
+        cc_keyboard.handle("cc_keyboard", cfg, cloud, [])
+        m_subp.assert_called_once_with(["setup-keymap", layout, variant])
+
```
Comment (TheRealFalcon) [inline tests/unittests/config/test_cc_keyboard.py:119]: Nit: Might be worth another test or two testing errors and ignores, but I'll leave that decision to you.

**Review by TheRealFalcon — CHANGES_REQUESTED** [review]
I left some comments inline, but overall good.

File: tests/unittests/config/test_cc_keyboard.py (hunk 2 above)
Comment (dermotbradley) [reply tests/unittests/config/test_cc_keyboard.py:140]: I added an additional testcase,test_alpine_linux_ignore_model.

I wasn't sure how to do testcases for errors as when I tried the error tests then failed.

File: cloudinit/distros/alpine.py (hunk 1 above)
Comment (dermotbradley) [reply cloudinit/distros/alpine.py]: done

**Review by TheRealFalcon — APPROVED** [review]
LGTM!

**Pull request merged**